finality-aleph/src/justification/requester.rs

```diff
@@ -23,7 +23,10 @@ const REPORT_THRESHOLD: u32 = 2;
 pub struct JustificationRequestStatus<B: BlockT> {
     block_number: Option<NumberFor<B>>,
     block_hash: Option<B::Hash>,
-    tries: u32,
+    block_tries: u32,
+    parent: Option<B::Hash>,
+    n_childred: Option<usize>,
```

Comment:
As a fellow parent I'm pretty sure you mean `child_dread`. ;>

Reply:
Fixed ;)